### DISCUSSION TITLE
Welcome to lineage_15.1_Installation_TSV Discussions!

### DISCUSSION THREAD

**Discussion by pgale** [pgale/lineage_15.1_Installation_TSV#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bingo-the-pyro** [comment]
Just popping to to say thanks for the excellent instructions!
I've custom ROMed multiple phones over the years, but this is a tad different, and this page succinctly summarizes other pages I have read. One TSV succesfully done, one to go :)

**Reply by pgale** [reply]
Thanks!

**Comment by OlivKoma** [comment]
How to install root? I can't give a root permission to any app :(

**Reply by BenJamesAndo** [reply]
I managed to root it. I wrote about it at https://community.home-assistant.io/t/my-take-on-a-home-dashboard-running-on-the-lenovo-thinksmart-view-and-managing-multiple-devices-running-fully-kiosk/718355/181?u=benando
